siac/main.go

```diff
@@ -160,6 +161,14 @@ func die(args ...interface{}) {
 	os.Exit(exitCodeGeneral)
 }
 
+// yesNo returns "Yes" if b is true, and "No" if b is false.
+func yesNo(b bool) string {
```

Comment:
There is probably a better name for this function. Maybe bToS? or just boolToString